Update README.md

cubeddu opened the pull request:
Issue #, if available: [#491](https://github.com/aws/graph-notebook/issues/491)
Description of changes:

**Update to README.md**
This PR removes the `--network="host"` option from the script to fix an issue where published ports are discarded when using host network mode. The `--network="host"` option is used to run a container in the host network mode, which allows the container to use the host's network stack and IP address. However, this option also causes published ports to be discarded, which can prevent the container from being accessible from outside the host.

Removing the `--network="host"` option from the script will run the container in the default bridge network mode, allowing published ports to be accessible from outside the host. This fix ensures that the container can be accessed from other machines on the network and resolves the issue of published ports being discarded.

This PR is a simple and effective solution to the issue and has been tested to ensure that it works as expected.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

cubeddu committed:
Update README.md

To resolve the problem of published ports being discarded in the host network mode, delete the "--network='host'" option.

xiazcy commented on docker/README.md:
````diff
@@ -6,10 +6,10 @@ Default password is `admin`.
 ## Example Runs
 ```sh
-docker run --network="host"  -p 8888:8888 -t graph-notebook
+docker run -p 8888:8888 -t graph-notebook
````
Do you know if there was any reason they added this `--network="host"` option as their example? Or is it potentially leftover code from older environments that used this flag?

xiazcy replied:
Looks we can add some explanation in the documentation around this? https://docs.docker.com/network/drivers/host/

cubeddu committed:
Update README.md

I have included extra documentation on the docker network mode and also added relevant local commands to make it easier to follow.
